### ISSUE TITLE
Labels of practice record types are not visible (permissions issue)

### ISSUE THREAD

**Issue by mstenta** [mstenta/farm_pcsc#20]
As a user with the TAP role, I can't currently see anything in the "Practice Type" column of the "Practices" View.

**Comment by paul121** [comment]
I started digging into in an implementation to use a new `access` handler. This would work for here but is not generalizable for all plan record types: https://github.com/mstenta/farm_pcsc/commit/7ac09668fa5cad12ca407679152566568837b3ae

In the meantime I've just added a custom views field that renders a nice practice type label as a link to the practice entity. This has various improvements over the out-of-the-box bundle label:

![Screenshot from 2024-05-31 15-48-25](https://github.com/mstenta/farm_pcsc/assets/3116995/f6f77586-366f-461c-93ec-f3ef03affe7f)

So... this is still an issue, but perhaps more of a core issue than this module?
